src/core/env/prod/terraform.tfvars

```diff
@@ -39,7 +39,7 @@ app_gateway_api_io_italia_it_certificate_name                     = "api-io-ital
 app_gateway_app_backend_io_italia_it_certificate_name             = "app-backend-io-italia-it"
 app_gateway_developerportal_backend_io_italia_it_certificate_name = "developerportal-backend-io-italia-it"
 app_gateway_api_io_selfcare_pagopa_it_certificate_name            = "api-io-selfcare-pagopa-it"
-app_gateway_min_capacity                                          = 4
+app_gateway_min_capacity                                          = 10
```

Comment:
to stay more safe... what do u think to increase to 15 or 20? only for first days